kubernetes-maven-plugin/doc/src/main/asciidoc/inc/build/_overview.adoc

```diff
@@ -13,7 +13,7 @@ Alternatively an external Dockerfile template or Docker archive can be used. Thi
 
 * *contextDir* specifies docker build context if an external dockerfile is located outside of Docker build context. If not specified, Dockerfile's parent directory is used as build context.
 * *dockerFile* specifies a specific Dockerfile path. The Docker build context directory is set to `contextDir` if given. If not the directory by default is the directory in which the Dockerfile is stored.
-* *dockerArchive* specifies a previously saved image archive to load directly. Such a tar archive can be created with `docker save` or the <<{plugin}:save>> goal. If a `dockerArchive` is provided, no `dockerFile` or `dockerFileDir` must be given.
+* *dockerArchive* specifies a previously saved image archive to load directly. Such a tar archive can be created with `docker save` or the <<jkube:save>> goal. If a `dockerArchive` is provided, no `dockerFile` or `dockerFileDir` must be given.
```

Comment:
Actually, this comes from DMP. In docker maven plugin we have `docker:save` goal

Reply:
This goal does not exist in JKube, what shall we do with the docs?

Reply:
Remove it?

Reply:
Not sure whether we have any plans to add JKube's own DMP in future